src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  padding: 100px 10px;
+  margin: 0 auto;
+  width: 726px;
+  gap: 1px;
+
+  @for $count from 1 through 31 {
+    &__day-#{$count} {
+      $root: &;
+      @include size($size);
+
+      display: flex;
+      align-items: center;
+      justify-content: center;
+
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+
+      background-color: $main-color;
+      cursor: pointer;
+      border: 1px solid #000;
+
+      transition-duration: 0.5s;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties